hash/megadriv.xml

```diff
@@ -7647,6 +7647,17 @@ Unsupported [Menacer] peripheral
 		</part>
 	</software>
 
+	<software name="sonic1jp2" cloneof="sonic">
+		<!--	Mod of rev. A present in Sonic Mega Collection, which changes the spike damage behavior to be like that of later titles.	-->
```

Comment:
`notes` field, also the extra spaces at head/tail triggers me.

Reply:
Done.